src/s3fs_util.cpp

```diff
@@ -1349,6 +1349,17 @@ void show_help ()
     "   There are many FUSE specific mount options that can be specified.\n"
     "   e.g. allow_other  See the FUSE's README for the full set.\n"
     "\n"
+    "Utility mode Options:\n"
+    "\n"
+    " -u, --incomplete-mpu   List incomplete multipart upload incomplete\n"
+    "                        objects and delete them all interactively.\n"
+    " --incomplete-mpu-list  List incomplete multipart upload incomplete\n"
+    "                        objects.\n"
+    " --incomplete-mpu-abort=number[Y|M|D|h|m]\n"
```

Comment:
Perhaps a simpler interface would be to just provide a day offset, e.g., abort any uploads older than 7 days?  It seems unlikely that users would want to abort downloads with smaller granularity.

Reply:
I forgot to write the "all" parameter.
In addition to specifying detailed parameters, there is an "all" parameter.
Let's use this "all" as the default value.